taca/storage/storage.py

```diff
@@ -229,11 +229,11 @@ def _send_to_swestore(f, dest, remove=True):
             misc.call_external_command('iput -K -P {file} {dest}'.format(file=f, dest=dest),
                     with_log_files=True)
             logger.info('Run {} sent correctly and checksum was okay.'.format(f))
+            if remove:
+                logger.info('Removing run'.format(f))
+                os.remove(f)
```

Comment:
Ahh ok! this `os.remove(f)` it is inside the method `_send_to_swestore(f)`, that will always me a compressed **file**, so `os.remove(f)` is okay here :-) :+1: